src/cascadia/TerminalApp/Resources/en-US/ContextMenu.resw

```diff
@@ -125,6 +125,10 @@
     <value>Terminal Dev</value>
     <comment>{Locked} The dev build will never be seen in multiple languages</comment>
   </data>
+  <data name="AppNameCan" xml:space="preserve">
+    <value>Terminal Canary</value>
+    <comment>{Locked=qps-ploc,qps-ploca,qps-plocm}</comment>

```

Comment:
We may need to clarify in the comment that "Canary" means "not the kind of bird, technically, but relating to an unstable or nightly build of a software product"